# How much am I saving (if anything) by using a Sodastream?

I think we need to think about the total cost per 12 oz. of soda. And let's use root beer as an example. We'll pit sodastream diet root beer against a can of diet A&W from Meijer.

## The Cost of Sodastream

Let: 

- $\hat{Y}$ denote total estimated cost
- $x$ be the quantity of sodastream root beer we wish to produce
- $a$ and $b$ be coefficients denoting the cost of syrup and $\mathrm{CO_2}$, respectively

$$\hat{Y}_{sodastream} = x\left(a_{syrup} + b_{\mathrm{CO_2}}\right)$$

### The Cost of Syrup

$$
   a_{syrup} = 
   \left(\textrm{355mL of soda}\right) 
   \times 
   \left(\frac{\$5.99}{\textrm{440mL syrup}}\right)
   \times
   \left( \frac{\textrm{15mL syrup}}{\textrm{240mL soda}} \right)
   $$

In [1]:
355 * (5.99 / 440) * (15 / 240)

0.3020525568181818

$$
a_{syrup} = \frac{\$0.3021}{\mathrm{355mL of soda}}
$$

### The cost of $\mathrm{CO_2}$

$$
b_{CO_2} = \left( \textrm{355mL of soda} \right)
           \left( 
                \left( 
                    \frac{\$29.99 - \left( \textrm{\$10 cylinder recycle discount} \right)}
                    {\mathrm{cylinder}}
                \right) 
           \right)
    \times 
    \left( \frac{\mathrm{cylinder}}{\textrm{60000mL of soda}} \right)
$$

In [3]:
355 * ((29.99 - 10) / 1) * (1 / 60000)

0.11827416666666667

$$
b_{\mathrm{CO_2}} = \frac{\$0.1183}{\textrm{355mL of soda}}
$$

### Total cost of a "can" of sodastream

$$
\hat{Y}_{sodastream} = 
    \textrm{355mL of soda}
    \times
    \left(
        \frac{\$0.3021 + \$0.1183}{\textrm{355mL of soda}}
    \right)
$$

In [4]:
0.3021 + 0.1183

0.4204

$$\textrm{sodastream price} = \frac{\$0.420}{\mathrm{can}}$$

### References

![sodastream exchange cylinder price, accessed 2015-11-02](sodastream-exchange-cylinder.png)

## Total cost of a can of diet A&W root beer

$$
    \left( \frac{\$11.00}{\textrm{4 cases}} \right)
    \times
    \left( \frac{\textrm{1 case}}{\textrm{12 cans}} \right)
$$

In [5]:
(11 / 4) * (1 / 12)

0.22916666666666666

$$
\textrm{can price} = \frac{\$0.229}{\textrm{can}}
$$

### References

![Cost of a 12-pack of cans of Diet A&W root beer, accessed 2015-11-02](diet-a-and-w.png)

# Conclusion

At just 23 cents per can, it seems drastically cheaper to buy A&W diet root beer (whose taste I prefer) to continuing to operate my sodastream (which costs 